X/Xorg_inputproto/build_tarballs.jl

```diff
@@ -4,6 +4,8 @@ using BinaryBuilder
 
 name = "Xorg_inputproto"
 version = v"2.3.2"
+# We bumped the version number because we converted the package to AnyPlatform
```

Comment:
The reason is a bit misleading, the real reason is because compat bounds changed.

Reply:
Updated.